static/elements/chromedash-userlist.js

```diff
@@ -55,6 +55,32 @@ class ChromedashUserlist extends LitElement {
     this.users = this.users.slice(0); // Refresh the list
   }
 
+  sortUsers() {
+    this.users.sort((a, b) => {
+      if ((a.is_admin && !b.is_admin) || (a.is_site_editor && (!b.is_site_editor && !b.is_admin))) {
+        return -1;
+      }
+      if ((b.is_admin && !a.is_admin) || (b.is_site_editor && (!a.is_site_editor && !a.is_admin))) {
+        return 1;
+      }
+      return a.email.localeCompare(b.email);
+    });
+  }
+
+  _onAdminToggle() {
```

Comment:
Both of your enhancement on this page are good.

Reply:
Thanks! 😊